cxx4opencl/address_spaces.txt

```diff
@@ -0,0 +1,235 @@
+// Copyright 2019 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[[opencl_c_variadic_macro]]
+=== Address spaces
```

Comment:
This document starts at a different "level" than the intro and acknowledgements document.  Is that intended?

Reply:
This is because this document contains subsections of into. 

```
== The {cpp} for OpenCL Programming Language 

This chapter documents various language features of {cpp} for OpenCL.
```

Although maybe there is a better way to structure this?

Reply:
> Although maybe there is a better way to structure this?

We can worry about this later - I just wanted to be sure it was intended.  Thanks!